packages/docs/package.json

```diff
@@ -1,92 +1,64 @@
 {
-  "name": "@availity/docs",
-  "version": "5.6.1",
-  "private": true,
-  "author": "Robert McGuinness <rob.mcguinness@availity.com>",
-  "description": "Storybook docs for availity-react",
-  "main": "index.js",
-  "scripts": {
-    "start": "start-storybook -p 2222 -c .storybook -s ./static",
-    "build": "build-storybook -c .storybook -s ./static -o ../../docs --quiet",
-    "deploy-docs": "storybook-to-ghpages",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
-  "keywords": [
-    "react",
-    "availity"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@github.com/Availity/availity-react.git"
-  },
-  "bugs": {
-    "url": "https://github.com/Availity/availity-react/issues"
-  },
-  "license": "MIT",
-  "devDependencies": {
-    "@availity/api-axios": "^3.0.1",
-    "@availity/api-core": "^3.0.0",
-    "@availity/app-icon": "^1.2.0",
-    "@availity/authorize": "^1.1.5",
-    "@availity/breadcrumbs": "^3.0.1",
-    "@availity/favorites": "^1.1.6",
-    "@availity/feature": "^1.1.5",
```

Comment:
Looks like all of these were removed when running latest `npm install`

Reply:
grr i didn't notice...will rollback changes and try again.